Src/EmbeddedScripts.CSharp.Roslyn.Scripting/ScriptingCodeBuilder.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using EmbeddedScripts.CSharp.Shared;
+using EmbeddedScripts.Shared;
+
+namespace EmbeddedScripts.CSharp.Roslyn.Scripting
+{
+    public class ScriptingCodeBuilder
+    {
+        public ScriptingCodeBuilder SetUserCode(string code)
+        {
+            Builder.SetUserCode(code);
+            return this;
+        }
+
+        public ScriptingCodeBuilder SetContainer(Container container)
+        {
+            Builder.SetContainer(container);
+            return this;
+        }
+
+        public string BuildCode() => 
+            Builder.BuildCode() + 
+            Environment.NewLine + 
+            $"new {Builder.ClassName}(Container).{Builder.MethodName}();";
```

Comment:
Hardcoded Container name

Reply:
This problem can solved using reflection. But do we really need in?

Reply:
Remind me why do you need to invoke it like that? Why not to just execute usercode as is (with prepended custom resolvers etc)?

Reply:
I thought we have to hide access to Container in user code. I will return the old version of user code invocation in the next commit.